examples/poem/mongodb/src/main.rs

```diff
@@ -60,13 +60,13 @@ async fn main() -> io::Result<()> {
     }
     tracing_subscriber::fmt::init();
 
-    let mongodb = Client::with_uri_str("mongodb://127.0.0.1:27017")
+    let mongodb = Client::with_uri_str("mongodb://0.0.0.0:27017")
```

Comment:
Connecting to 0.0.0.0 isn't strictly speaking valid. (But, in practice, it should be equivalent to connecting to localhost.)